repostats/repostats.py

```diff
@@ -1090,9 +1092,20 @@ def get_astroconda_membership(name="", data=""):
     return False
 
 
-def scrap_rtd_badge(repoorg, reponame):
-    import base64
-    import re
+def scrape_rtd_badge(org=None, name=None):
```

Comment:
you were too fast, I agreed with you ;)  Some of the other functions allow either url or org+name so they don't have required, but here you need both, so I agree, you don't have to set to None. Do you want to change back?

Reply:
Naw, it's fine for this PR. Clean up can be a separate PR. :D